docs/Configuration.md

```diff
@@ -369,6 +369,22 @@ You can collect coverage from those files with setting `forceCoverageMatch`.
 }
 ```
 
+### `freezeCoreModules` [boolean]
+
+Default: `false`
+
+Prevents overriding node's core module methods so to prevent memory leaks. 
+
+Attempt to override such a property will fail silently.
+
+Running with [`verbose`](#verbose-boolean) flag will enable informative print outs.
+
+### `freezeCoreModulesWhitelist` [Array<string>]
+
+Default: `['crypto']`
```

Comment:
Proxies have issues with bound native methods, hence `crypto.randomBytes` will fail. Tried to hack it from multiple directions with no success.